modules/lang-mustache/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module org.elasticsearch.mustache {
+    requires org.elasticsearch.base;
+    requires org.elasticsearch.server;
+    requires org.elasticsearch.xcontent;
+    requires org.apache.logging.log4j;
+    requires com.github.mustachejava;
+
+    exports org.elasticsearch.script.mustache;
```

Comment:
Where is the export needed? Is this only needed by server for reflective construction?

Reply:
The HLRC uses types from `org.elasticsearch.script.mustache`, see `org.elasticsearch.client.RequestConverters`.

Reply:
Ok, perhaps add a TODO to remove this once HLRC is removed (it's dead code, just used by tests now)